doc/self-hosted/index.md

```diff
@@ -115,5 +115,29 @@ OPTIONS
   ```
   *Important:* For more details about Apache reverse proxy configuration checkout the [documentation](https://httpd.apache.org/docs/current/mod/mod_proxy.html) - especially the [Securing your Server](https://httpd.apache.org/docs/current/mod/mod_proxy.html#access) section
   
+  ### Start code-server as systemd unit
+  Create `/lib/systemd/system/code-server.service` with the following contents:
+  ```
+  [Unit]
+  Description=Code Server IDE
+  After=network.target
+
+  [Service]
+  Type=simple
+  User=<USER>
```

Comment:
Please also set environment file. There is a instance the User environment is not being used by `systemd`.

Reply:
Please suggest changes as I am not aware of the issue.

Reply:
```suggestion
  User=<USER>
  EnvironmentFile=$HOME/.profile
```

Reply:
For reference, see my [systemd Unit file](https://gist.github.com/sr229/9600b26d9eada8133a8587ce05d846a9)

Reply:
@sr229 The Environment file can't work it has wrong formart systemd expects a other formart it will not execute that file with if else statements.

```
VAR=VALUE
```

is the only thing that is accepted in it!